.github/workflows/node.js.yml

```diff
@@ -17,3 +17,7 @@ jobs:
       - run: yarn workspace frontend build
       - run: yarn workspace discord-bot lint
       - run: yarn workspace discord-bot build
+      - run: echo "${{secrets.ROOT_ENV_TESTING }}" > .env
+      - run: echo "${{secrets.API_ENV_TESTING }}" > packages/api/.env
+      - run: yarn mongodb:start
+      - run: yarn workspace api test
```

Comment:
I'm wondering if we really need to run this test on every PR. We'd need to run this only when the API is altered, right?

Reply:
yeah good point -- I should probably split up the github actions to only run relevant checks for modified workspaces.

Reply:
done